ui/cypress/e2e/dashboardsView.test.ts

```diff
@@ -73,15 +73,43 @@ describe('Dashboard', () => {
     cy.getByTestID('empty-state').should('exist')
 
     const noteText = 'this is a note cell'
+    const headerPrefix = '#'
 
     // Note cell
     cy.getByTestID('add-note--button').click()
     cy.getByTestID('note-editor--overlay').within(() => {
-      cy.get('.CodeMirror').type(noteText)
+      cy.get('.CodeMirror').type(`${headerPrefix} ${noteText}`)
+      cy.getByTestID('note-editor--preview').contains(noteText)
```

Comment:
```suggestion
      cy.contains(noteText)
```
i think this is a less failure prone pattern - i'm pretty sure anytime you do `getByTestID().contains()` you can just do `.contains()`. using `contains` rather than abstracts the need to know the underlying structure or test ids away from you and lets cypress handle getting the node.

Reply:
for sure.  for this, i specifically want to check the "preview" box and not the text area.